packages/cubejs-playground/src/components/QueryTabs/QueryTabs.tsx

```diff
@@ -220,6 +232,13 @@ export function QueryTabs({
     saveTabs({ activeId, tabs });
   }
 
+  function handleDrilldownModalClose() {
+    const MODAL_ANIMATION_TIME = 300; // ms
+    setTimeout(() => {
```

Comment:
This `setTimeout` delays clearing the drilldown query a bit until the modal closes. Otherwise it will close immediately which is not really nice.